orchestration/hca_orchestration/solids/copy_project/data_file_ingestion.py

```diff
@@ -74,8 +76,18 @@ def _generate_control_file(context: AbstractComputeExecutionContext,
     ingest_items = []
     for data_entity in data_entities:
         file_bucket_and_prefix = parse_gs_path(data_entity.path)
+        bucket = Bucket(storage_client, file_bucket_and_prefix.bucket)
```

Comment:
Working around an issue where the TDR SA cannot read from the source dataset's bucket any longer.